src/platform_impl/windows/window.rs

```diff
@@ -99,14 +131,11 @@ impl Window {
                             }
                         }),
                     );
-                    let handler_interface_ptr =
-                        &mut (*file_drop_handler.data).interface as LPDROPTARGET;
 
-                    assert_eq!(
-                        ole2::RegisterDragDrop(win.window.0, handler_interface_ptr),
-                        S_OK
-                    );
-                    Some(file_drop_handler)
+                    let drop_target = IDropTarget::from(file_drop_handler);
```

Comment:
Hmm the `cast()` function provided by `#[implement]` is private.

Reply:
@kennykerr is this how I'm supposed to do it?
I have now idea if I'm using the type with the implement macro right...
Is there a better way?